js/index.js

```diff
@@ -1 +1,57 @@
 // Your code goes here
+
+const title = document.querySelector("header h1");
+const busImg = document.querySelector(".home img")
+const mapImg = document.querySelector(".img-content")
+const inverseMapImg = document.querySelector(".inverse-content .img-content")
+const text = document.querySelector(".content-section p")
+const welcomeP = document.querySelector("header p")
```

Comment:
Good selection here, but don't forget about nav items! They make easy targets for event listeners.